### PR TITLE
fix: honor DET_MASTER_CERT_NAME with shells

### PR DESCRIPTION
**Pull request by rb-determined-ai**
## Description

Tested manually.

### CODE REVIEW

File: cli/determined_cli/tunnel.py [hunk 1]
```diff
@@ -129,14 +139,11 @@ def http_connect_tunnel(master: str, service: str, master_cert: Optional[str]) -
...
-            "usage: {} MASTER_ADDR SERVICE_UUID [MASTER_CERT]".format(sys.argv[0]), file=sys.stderr
-        )
-        sys.exit(1)
-
-    master = sys.argv[1]
-    service = sys.argv[2]
-    master_cert = None if len(sys.argv) < 4 else sys.argv[3]
-
-    http_connect_tunnel(master, service, master_cert)
+    parser = argparse.ArgumentParser(description="Tunnel through a Determined Master")
```
Comment (dzhu) [inline cli/determined_cli/tunnel.py:142]:
```suggestion
    parser = argparse.ArgumentParser(description="Tunnel through a Determined master")
```

File: cli/determined_cli/tunnel.py [hunk 2]
```diff
@@ -129,14 +139,11 @@ def http_connect_tunnel(master: str, service: str, master_cert: Optional[str]) -
...
+    parser.add_argument("master_addr", type=str)
+    parser.add_argument("service_uuid", type=str)
+    parser.add_argument("--cert-file", type=str, nargs="?")
+    parser.add_argument("--cert-name", type=str, nargs="?")
```
Comment (dzhu) [inline cli/determined_cli/tunnel.py:146]: non-blocking: `nargs` isn't doing anything for us, is it? The options are optional, but the argument may as well be required if the option is present.

**Review by dzhu — APPROVED** [review]

File: cli/determined_cli/tunnel.py (hunk 1 above)
Comment (rb-determined-ai) [reply cli/determined_cli/tunnel.py:142]: done

File: cli/determined_cli/tunnel.py (hunk 2 above)
Comment (rb-determined-ai) [reply cli/determined_cli/tunnel.py:146]: Guess not.  Neither is `type=str`, looks like.

**Pull request merged**